Makefile

```diff
@@ -7,7 +7,7 @@ UTILS ?= utils
 .PHONY: setup-venv setup clean-pyc clean-test test mypy lint docs check
 
 setup-venv:
-	$(HIDE)python3.12 -m venv $(VENV)
+	$(HIDE)python -m venv $(VENV)
```

Comment:
On my system (Macbook) there is no "python" anymore.  There is only "python3".  I'm using "python3.12".  I think for development it's good to settle on a specific version of python so that we know code we write will work on each others' systems.  What happened when you ran this?  Could it not find python3.12?  Do you have a python3 installed (maybe an earlier version)?

I'm afraid that "python" runs "python 2.x", which we want to avoid.  Although, if that was the case I don't think "pip3" would work, because I don't think pip3 exists in python 2.x.